css/_tooltips.scss

```diff
@@ -0,0 +1,224 @@
+.yoast-tooltip {
+	position: relative;
+}
+
+.yoast-tooltip::after {
+	display: none;
+	position: absolute;
+	z-index: 1000000;
+	padding: 5px 8px;
+	border-radius: 3px;
+	opacity: 0;
+	color: #fff;
+	background: rgba(0,0,0,0.8);
+	text-shadow: none;
+	font: normal normal 11px/1.5 Helvetica, arial, nimbussansl, liberationsans, freesans, clean, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+	text-align: center;
+	white-space: pre;
+	text-decoration: none;
+	letter-spacing: normal;
+	text-transform: none;
+	word-wrap: break-word;
+	content: attr(aria-label);
+	pointer-events: none;
+
+	-webkit-font-smoothing: subpixel-antialiased;
+}
+
+.yoast-tooltip::before {
+	display: none;
+	position: absolute;
+	z-index: 1000001;
+	width: 0;
+	height: 0;
+	border: 5px solid transparent;
+	opacity: 0;
+	color: rgba(0,0,0,0.8);
+	content: "\00a0";
+	pointer-events: none;
+}
+
+@-webkit-keyframes yoast-tooltip-appear {
```

Comment:
This isn't necessary because the grunt-task `auto-prefixer` will add prefixes on `build:css`